src/app/pages/room/room.page.ts

```diff
@@ -34,6 +35,7 @@ export class RoomPage implements OnDestroy {
       private readonly route: ActivatedRoute,
       private readonly clueService: ClueService,
       private readonly utilService: UtilService,
+      private readonly soundService: SoundService,
```

Comment:
this needs to be public
```suggestion
      public readonly soundService: SoundService,
```